php-nginx/usr/local/share/env/55-stack

```diff
@@ -0,0 +1,6 @@
+DEFAULT_COMPOSER_FLAGS="--no-interaction --optimize-autoloader"
```

Comment:
they don't for sourcing, but for format detection they are needed, so I've added them